document/01-intro/index.tex

```diff
@@ -0,0 +1,34 @@
+\section{System architecture}\label{sec:system-architecture}
+
+The Papaya software consists of three essential parts.
+Those are:
+\begin{itemize}
+    \item Papaya Keycloak --- a Keycloak authentication server, it provides
+    authentication, authorization and user management services;
```

Comment:
Ja w sumie mam instynkt, żeby używać Oxford comma i tutaj (i jeszcze gdzieś tam) bym dała. Generalnie to jest _kontrowersyjne_ i _nieistotne_, ale głos serca kazał mi napisać

Reply:
Podpisuję się pod tą petycją. Choć spotkałem się również z koncepcją średnikowania na końcu podpunktu.